[PATCH] Fix build issue with musl

arpa/nameser.h doesn't use the same macro name to avoid several
inclusions.

Finally had an issue with framebuffer_service.c since it was missing the
TEMP_FAILURE_RETRY macro.

Signed-off-by: Gary Bisson <gary.bisson@boundarydevices.com>
---
 core/adbd/arpa_nameser.h              | 12 +++++++++---
 core/adbd/base64.c                    |  1 -
 core/adbd/framebuffer_service.c       |  1 +
 core/adbd/qemu_pipe.h                 |  1 -
 core/include/cutils/android_reboot.h  |  8 ++++++--
 core/include/cutils/bitops.h          | 10 ++++++----
 core/include/cutils/partition_utils.h |  8 ++++++--
 extras/ext4_utils/sha1.c              |  3 ---
 extras/ext4_utils/sha1.h              | 13 ++++++-------
 9 files changed, 34 insertions(+), 23 deletions(-)

diff --git a/core/adbd/arpa_nameser.h b/core/adbd/arpa_nameser.h
index 438dc04..b2a28d6 100644
--- a/core/adbd/arpa_nameser.h
+++ b/core/adbd/arpa_nameser.h
@@ -52,6 +52,8 @@
 
 #ifndef _ARPA_NAMESER_H_
 #define _ARPA_NAMESER_H_
+#ifndef _ARPA_NAMESER_H
+#define _ARPA_NAMESER_H
 
 #define BIND_4_COMPAT
 
@@ -574,4 +579,5 @@ __END_DECLS
 #define  XLOG(...)   do {} while (0)
 #endif
 
+#endif /* !_ARPA_NAMESER_H */
 #endif /* !_ARPA_NAMESER_H_ */
diff --git a/core/adbd/framebuffer_service.c b/core/adbd/framebuffer_service.c
index 20c08d2..48e0241 100644
--- a/core/adbd/framebuffer_service.c
+++ b/core/adbd/framebuffer_service.c
@@ -26,6 +26,7 @@
 #include "fdevent.h"
 #include "adb.h"
 
+#include <cutils/fs.h>
 #include <linux/fb.h>
 #include <sys/ioctl.h>
 #include <sys/mman.h>
-- 
2.6.1
